src/interfaces/dissolution-confirmation-interface.ts

```diff
@@ -4,4 +4,6 @@ export interface DissolutionConfirmationResourceIF {
   act: string
   modalTitle: string
   confirmButtonText:string
+  additionalLabel: string
```

Comment:
use like additionalLabel?: string,  I think this will optional field for Sp/GP,
same for dissolutionType

Reply:
resolved